### DISCUSSION TITLE
Alert about "Duplicated string found in the file."

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10902] [Q&A] [answered]
![image](https://github.com/WeblateOrg/weblate/assets/11861496/8a9553aa-e653-43c8-883d-b63efc0babd1)

I got this alert mail about duplicated strings. I don't understand the reason and can not see a problem.
The project is here: https://translate.codeberg.org/projects/backintime/

What does Weblate mean by "file"?
How can I solve it? "Please fix this by removing duplicated strings with same identifier from the translation files." does not help. Because it is the glossary. I don't use any files related to the Glossary. It is created "automatically" by Weblate itself.

**Comment by nijel** [comment]
Most likely this is caused by https://github.com/WeblateOrg/weblate/issues/9701

**Reply by buhtz** [reply, marked as answer]
To prevent other readers from following links because the hastag-numbers do not indicate if it is an Issue/PR/Discussion/AnotherFancyFeature.
It is caused by known bug. ;)
